static/app/views/settings/project/navigationConfiguration.tsx

```diff
@@ -39,8 +39,8 @@ export default function getConfiguration({
         },
         {
           path: `${pathPrefix}/tags/`,
-          title: t('Tags'),
-          description: t("View and manage a  project's tags"),
+          title: t('Tags & Context'),
+          description: t("View and manage a project's tags and highlights"),
```

Comment:
Is it okay to mention this even without the feature flag enabled?

Reply:
good call, replaced with `and context`, thanks!